apache/superset
# fix(alert modal): set current user as default alert owner during new alert initialization

### SUMMARY
Currently, when the modal window opens, the "owners" field is left unpopulated. This is slightly annoying as it would be more user friendly and efficient to automatically pre-populate the field with the currently logged-in user.

To address this, I propose retrieving the current user from Redux state and setting it as the default owner when the alert modal is opened.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:
<img width="1166" alt="Screenshot 2023-05-15 at 1 43 35 AM" src="https://github.com/apache/superset/assets/97945303/bf0a361b-c4d5-47fd-a391-4dd95643f13a">

After:
<img width="1167" alt="Screenshot 2023-05-15 at 1 43 57 AM" src="https://github.com/apache/superset/assets/97945303/258662d3-a135-4098-889e-a4f8dde0e5f3">


### TESTING INSTRUCTIONS
Open Superset in browser > Click on the "Settings" menu on top right > Click on "Alerts & Reports" > on the "Alerts & Reports" page, click on the "+ ALERT" button to create a new alert & notice the behavior.

### ADDITIONAL INFORMATION
NA
